sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -25,13 +25,18 @@
     _SUBSCRIPTION_ID_ENV_VAR.split("+")[0] if _SUBSCRIPTION_ID_ENV_VAR else None
 )
 _logger = logging.getLogger(__name__)
+_logger.propagate = False
+_logger.setLevel(logging.INFO)
 _DIAGNOSTIC_LOG_PATH = _get_log_path()
+_ATTACH_SUCCESS_DISTRO = "4200"
+_ATTACH_SUCCESS_CONFIGURATOR = "4201"
```

Comment:
What is the logic behind choosing these specific msgIds in the range? Should we have some sort of guideline in how we choose them?

Reply:
We're free to choose whatever, we could do 4000-4003 for these, but I figured it could be useful to actually organize them like status codes:
41XX: information
42XX: Success
44XX: Failure

Open to another approach.

Reply:
Interesting. Yeah that looks good to me.